data/entities.json

```diff
@@ -833,5 +833,41 @@
     "homepage": "https://usabilla.com",
     "categories": ["analytics"],
     "domains": ["w.usabilla.com"]
+  },
+  {
+    "name": "Fastly Insights",
+    "homepage": "https://insights.fastlylabs.com",
+    "categories": ["analytics"],
+    "domains": ["www.fastly-insights.com"]
+  },
+  {
+    "name": "Visual IQ",
+    "homepage": "https://www.visualiq.com",
+    "categories": ["analytics"],
+    "domains": ["t.myvisualiq.net"]
+  },
+  {
+    "name": "Snapchat",
```

Comment:
@patrickhulce do you want to call this something else? It's snapchats tracking pixel.

Reply:
this works for me :)